README.md

```diff
@@ -40,64 +31,144 @@ set `ZSH_THEME="geometry"` in your `.zshrc`.
 
 Add `zplug "frmendes/geometry"` to your `.zshrc`
 
-### Dependencies
+## Dependencies
+
+The symbol for rebasing comes from a [Powerline patched font](https://github.com/powerline/fonts). If you want to use it, you're going to need to install one from the font repo. The font used in the screenshots is [Roboto Mono](https://github.com/powerline/fonts/tree/master/RobotoMono). You can also try to [patch it yourself](https://github.com/powerline/fontpatcher).
+
+You can also change the rebase symbol by setting the `GEOMETRY_SYMBOL_GIT_REBASE` variable.
+
+## What it does
+
+To allow a pleasant configuration and customization, geometry works with the concept of plugins.
+
+In a nutshell, it can:
+
+- give you a custom, colorizable prompt symbol
+- change the prompt symbol color according to the last command exit status
+- make the prompt symbol color change with your hostname
+- display current git branch, state and time since last commit
+- tell you whether you need to pull, push or if you're mid-rebase
+- display the number of conflicting files and total number of conflicts
+- display the running time of long running commands
+- set the terminal title to current command and directory
+- make you the coolest hacker in the whole Starbucks
+
+The right side prompt is print asynchronously, so you know it's going to be
+fast™.
+
+## Plugins
+
+geometry has an internal plugin architecture. The default plugins are `exec_time` and `git`. But you can also enable a number of them just by setting the `GEOMETRY_PROMPT_PLUGINS` variable in your own configuration files:
```

Comment:
I'm gonna take a look at this [here](https://github.com/frmendes/geometry/issues/55).